airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/github.py

```diff
@@ -17,6 +37,59 @@ def github_connector_folders(context):
     return Output(folder_names, metadata={"preview": folder_names})
 
 
+@asset(required_resource_keys={"github_connector_repo", "github_connectors_metadata_files"}, group_name=GROUP_NAME)
+def github_metadata_file_md5s(context):
+    """
+    Return a list of all the folders in the github connectors directory.
+    """
+    github_connector_repo = context.resources.github_connector_repo
+    github_connectors_metadata_files = context.resources.github_connectors_metadata_files
+
+    metadata_file_paths = {
+        metadata_path: _get_md5_of_github_file(context, github_connector_repo, metadata_path)
+        for metadata_path in github_connectors_metadata_files
+    }
+
+    return Output(metadata_file_paths, metadata={"preview": metadata_file_paths})
+
+
+@asset(required_resource_keys={"latest_metadata_file_blobs"}, group_name=GROUP_NAME)
+def stale_github_metadata_file(context, github_metadata_file_md5s: dict) -> OutputDataFrame:
```

Comment:
```suggestion
def stale_gcs_latest_metadata_file(context, github_metadata_file_md5s: dict) -> OutputDataFrame:
```
As the source of truth is the github repo, isn't it a "latest" metadata that can be stale?

Reply:
good callout!